src/hint_processor/builtin_hint_processor/builtin_hint_processor_definition.rs

```diff
@@ -343,13 +343,16 @@ impl HintProcessor for BuiltinHintProcessor {
             }
             hint_code::IS_ZERO_NONDET => is_zero_nondet(vm, exec_scopes),
             hint_code::IS_ZERO_ASSIGN_SCOPE_VARS => is_zero_assign_scope_variables(exec_scopes),
-            hint_code::DIV_MOD_N_PACKED_DIVMOD => div_mod_n_packed_divmod(
-                vm,
-                exec_scopes,
-                &hint_data.ids_data,
-                &hint_data.ap_tracking,
-            ),
-            hint_code::DIV_MOD_N_SAFE_DIV => div_mod_n_safe_div(exec_scopes),
+            hint_code::DIV_MOD_N_PACKED_DIVMOD_V1 | hint_code::DIV_MOD_N_PACKED_DIVMOD_V2 => {
```

Comment:
In the int_code::DIV_MOD_N_PACKED_DIVMOD_V2 case, the N constant should be searched in the scope